src/devtools/client/debugger/src/components/Editor/LineHitCounts.module.css

```diff
@@ -3,17 +3,15 @@
   align-items: center;
   justify-content: center;
 
-  width: 26px;
-  height: var(--theme-code-font-size);
-  margin: 2px;
-  padding-top: 1px;
+  width: 100%;
+  height: 16px;
```

Comment:
This was using a CSS var before because we have a "large text size" experimental setting. I don't think a numeric value like 16px is going to work for that.

Reply:
Oops, forgot about that, I'll get this added back!